_includes/docs/user-guide/rpc.md

```diff
@@ -82,6 +82,36 @@ This token should belong to either
  
 You can use the following [guide](/docs/{{docsPrefix}}reference/rest-api/#rest-api-auth) to get the token.
 
+## Persistent RPC
+
+ThingsBoard provides a new and unique feature: **Persistent RPC**. 
+The difference between Basic RPC and Persistent RPC is that the second has an increased timeout and is _permanently_ stored in the Database.
```

Comment:
The word "permanently" means that RPC command is stored indefinitely. This is wrong. The command is stored for a configurable time.